clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -163,10 +173,10 @@ public void run() {
      * @param now
      *            The current POSIX time in milliseconds
      */
-    public void run(long now) {
+    protected void run(long now) {
```

Comment:
Yes. This method seems exposed only for unit test purpose. So protected seems good enough. Is there any other case we need access to this method?